.changes/next-release/Feature-acb30000-5eeb-4a1a-85ce-1df104b9cc86.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Security issue hover telemetry includes additional metadata"
```

Comment:
No need for a changelog item since this change is not user facing

Reply:
sounds good, removing it